.github/workflows/agent-refresh.yml

```
@@ -0,0 +1,541 @@
+          git config user.email "github-actions[bot]@users.noreply.github.com"
+          git add images/vercel
+          git commit -m "chore(vercel): refresh coding-agent versions
+
+          Updates images/vercel/agents.json and the matching provenance
+          version records to the registry-latest versions validated by this
+          run's candidate build and dual smoke gates."
+          git push --force origin HEAD:agent-update/agents
+          echo "Pushed ${GITHUB_SHA:0:12}... -> agent-update/agents"
+
+      - name: Create or update the promotion PR
+        if: steps.skip.outputs.skip != 'true'
```

actionlint (1.7.12)

[error] property "skip" is not defined in object type {candidate: {conclusion: string; outcome: string; outputs: {string => string}}; drift: {conclusion: string; outcome: string; outputs: {string => string}}; manifest: {conclusion: string; outcome: string; outputs: {string => string}}; provenance: {conclusion: string; outcome: string; outputs: {string => string}}; redact_final: {conclusion: string; outcome: string; outputs: {string => string}}; redact_publisher: {conclusion: string; outcome: string; outputs: {string => string}}}

(expression)